### ISSUE TITLE
JetMenu loads the font-awesome library, the same as Elementor. Need to disable it

### ISSUE THREAD

**Issue by dislokacia** [CrocoBlock/suggestions#2321]
JetMenu loads the font-awesome library, the same as Elementor. Need to disable it

**Comment by stasonua0** [comment]
+1

**Comment by XVII** [comment]
Yes, I also want to completely disable **fa-brands** as I don't use those icons and they're the largest set (76kb). I already dequeue this via Elementor `wp_deregister_style( 'elementor-icons-fa-brands')`.

**Comment by XVII** [comment]
Perhaps removing all duplicate font-awesome and leveraging Elementor's already included Icon Library is the cleaner answer (#1284)

**Comment by jeffreysfridge** [comment]
I would also like an option to disable JetMenu's font awesome library and just use the pro one I loaded in with Elementor.

**Comment by JanWebofka** [comment]
Hello probably same problem i disabled all Font-awesome and uset custom icons added to media library and menu doesnt work. 
![image](https://github.com/user-attachments/assets/d74c98fd-92cf-4d98-b518-6b39aa18d868)
